iceoryx_posh/etc/iceoryx/roudi_config_medium_default.toml

```diff
@@ -0,0 +1,32 @@
+[general]
```

Comment:
why a new file? just adjust the old one. If somebody needs a custom config it is easy to create one.

Also, I wouldn't copy the file to /etc/iceoryx/roudi_config.toml but /etc/iceoryx/roudi_config.toml.example and hardcode sane defaults in the code. If a user needs a custom config he/she can just copy the file to /etc/iceoryx/roudi_config.toml and that one would be used. Just have a look how other deamons handle this.